test/functional/tools/sam_to_bam_native.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="sam_to_bam_native_conversion" name="Test sam to bam native conversion">
+    <command><![CDATA[
+        cat '$input1' > '$outfile1'
+    ]]>
+    </command>
+    <inputs>
+        <param name="input1" type="data" format="bam" label="Concatenate Dataset"/>
```

Comment:
Wouldn't this test use the `sam_to_bam` converter ?
Maybe change the input format to `bam_native` ?
If we add this to test/functional/tools/samples_tool_conf.xml it should be run during the functional tests.

Reply:
I fixed this test, it should test the conversion from SAM to BAM (unsorted).
Thanks @mvdbeek for the review.